components/Register.jsx

```diff
@@ -1,7 +1,49 @@
 import React from "react";
+import Image from "next/image";
 
 const Register = () => {
-  return <div>Register</div>;
+  return (
+    <div className="font-bold w-[45%] h-[60%] m-4">
+      <container className=" bg-acm-blue rounded-[10%] grid">
+        <div
+          style={{ paddingTop: "12%" }}
+          className="text-acm-black text-center font-bold"
+        >
+          <h3 style={{ fontSize: "3vw" }}>
```

Comment:
Same as comment above, try to style using tail wind

ex: text-[3vw]

* also haven't checked if font will overflow on large screens but something to keep in mind